### ISSUE TITLE
On second monitor line rezises fast

### ISSUE THREAD

**Issue by afk-mario** [Centril/firefox-line#10]
On my second monitor when a page loads firefox-line gets reized really fast.

**Comment by Centril** [comment]
How do you mean by "resized really fast"?
Also, is it specific to your second monitor?

**Comment by afk-mario** [comment]
I will try to record a GIF when it happens again, it happens on Windows and mac Firefox 49.0.1 in both monitors, but it's more common on the second.

**Comment by afk-mario** [comment]
![firefox](https://cloud.githubusercontent.com/assets/1025751/19027481/839cd516-88f6-11e6-8ddf-a6e16d5e1eec.gif)

**Comment by Centril** [comment]
Right, this is a known issue for me - it happens when there are just enough tabs that the tab-scroller can't decide if it's too many tabs or not...

Will have to take another look at it and see if I can solve it somehow...

**Comment by Centril** [comment]
iirc, this only happens if you have pinned tabs too.

The relevant functionality is controlled here: https://github.com/Centril/firefox-line/blob/master/src/main.js#L277-L353

One possible "hack" to fix it is to track the seconds between this happening, and if it's more than a threshold, then the scroller is used...
